python/pyhail/keytable.py

```diff
@@ -301,3 +359,26 @@ def select(self, field_names):
             return KeyTable(self.hc, self.jkt.select(field_names, new_key_names))
         except Py4JJavaError as e:
             self._raise_py4j_exception(e)
+
+    def toDF(self, expand=True, flatten=True):
```

Comment:
Is there a reason this method name is camel case and others are not (example: ``key_by``)